package.json

```diff
@@ -25,27 +25,31 @@
   },
   "types": "dist/lib/index.d.ts",
   "scripts": {
-    "build": "npm run build:assets && npm run build:css && npm run build:lib && npm run build:client",
+    "build": "npm run build:assets && npm run build:css && npm run build:lib && npm run build:client && npm run build:client:react",
```

Comment:
Should we leave `npm run build:client:react` out of this script for now, or add an exclusion for `dist/react` to the `files` property? (Since tests aren't in yet – I know `exports` has been left off but it'll still end up in the package otherwise.)